trunk/source/WitnessParser/src/de/uni_freiburg/informatik/ultimate/witnessparser/YamlWitnessParser.java

```diff
@@ -65,115 +62,61 @@
  * @author Frank Schüssele (schuessf@informatik.uni-freiburg.de)
  */
 public class YamlWitnessParser {
-
-	public static Witness parseWitness(final String yamlInput) throws IOException {
-		final YamlInput witnessInput = Yaml.createYamlInput(yamlInput);
-		return parseWitnessEntries(witnessInput);
-	}
-
 	public static Witness parseWitness(final File yamlInput) throws IOException {
-		final YamlInput witnessInput = Yaml.createYamlInput(yamlInput);
-		return parseWitnessEntries(witnessInput);
-	}
-
-	private static Witness parseWitnessEntries(final YamlInput witnessInput) throws IOException {
-		final YamlSequence witnessEntries = witnessInput.readYamlSequence();
-		final List<WitnessEntry> entries = new ArrayList<>();
-
-		for (final YamlNode witnessEntry : witnessEntries) {
-			final WitnessEntry newEntry = parseWitnessEntry(witnessEntry);
-			entries.add(newEntry);
-		}
-
-		return new Witness(entries);
+		final LoaderOptions loaderOptions = new LoaderOptions();
+		loaderOptions.setCodePointLimit(30 * 1024 * 1024);
```

Comment:
Is there a specific reason for this number? (And can we make it a `final static private` constant?)

Reply:
Sure, I thought I already did that.